src/pages/projectSettings/tabs/NotificationsTab/getFormSchema.tsx

```diff
@@ -71,7 +123,9 @@ export const getFormSchema = (
         },
       },
       subscriptions: {
-        "ui:placeholder": "No subscriptions are defined.",
+        "ui:placeholder": (
+          <Placeholder>No subscriptions are defined.</Placeholder>
```

Comment:
It would probably always be desirable to have this spacing applied to the 0 items placeholder, so we could update it here instead of adding another tag

https://github.com/evergreen-ci/spruce/blob/2929726b7f156f5898849f6ea980b7aa75857e76/src/components/SpruceForm/FieldTemplates.tsx#L303